content/docs/intro/concepts/stack.md

```diff
@@ -28,9 +30,13 @@ If you are using Pulumi in your organization, by default the stack will be creat
 $ pulumi stack init broomllc/staging
 ```
 
-Fully qualified stack names also include the project name, in the form `orgName/projectName/stackName`, and this fully-qualified format is required in some contexts.  In most contexts, the shorthands `orgName/stackName` or `stackName` are valid and use the default organization and the current project context.
+Fully qualified stack names also include the project name, in the form `orgName/projectName/stackName`, and this fully-qualified format is required in some contexts. In most contexts, the shorthands `orgName/stackName` or `stackName` are valid and use the default organization and the current project context.
+
+{{% notes "info" %}}
+Note that while stacks with applied configuration settings will often be accompanied by `Pulumi.<stack-name>.yaml` files, these files are not created by `pulumi stack init`. They are created and managed with [`pulumi config`]({{< relref "/docs/reference/cli/pulumi_config" >}}).
```

Comment:
Something about this is causing an odd line break between the icon and the text:

<img width="291" alt="image" src="https://user-images.githubusercontent.com/274700/106214901-a2cf3d00-6184-11eb-8f93-f05e7e800b28.png">

Reply:
I pulled the sentence in that was originally after the line break and that fixed the problem.